src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1796,42 +1796,26 @@ private BaseListSyntax ParseBaseList()
             try
             {
                 // first type
```

Comment:
If I understood correctly, the code that was removed (here and below) handled the case `class C : where T : ...` and `class C : I1, where T : ...`.
Do we have parser tests for those? If not, could you add them?